### ISSUE TITLE
Document qbsp flags needed?

### ISSUE THREAD

**Issue by elliottt** [jitspoe/godot_bsp_importer#6]
This project is so neat!

I'm pretty new to TrenchBroom and the bsp workflow in general, and hit a bit of a stumbling block with mapping textures to materials in godot: it wasn't obvious to me initially that I needed to pass the `-path` flag to `qbsp`, so that it would know where to find my textures. I'm not sure what the best way to document this would be, and perhaps people that are more familiar with the toolchain just know to do this already, but it could help others have an easier time with their initial experiments if there was an explanation of how to export successfully from TrenchBroom.

Thanks again for this plugin, it was super gratifying to see my first experiments with TrenchBroom load into Godot :)

**Comment by Skaruts** [comment]
This would be quite nice. 

I'm not getting good results. I'm passing `-bsp2` and `-path` (thanks to you). Are there other arguments I should be passing?

**Comment by jitspoe** [comment]
Yeah, I want to do a tutorial on how to set this stuff up, but I haven't gotten around to it.  Time keeps running away from me!

I wouldn't use -bsp2, as I don't think that format is 100% supported correctly.  It's only necessary when you go beyond certain limits.

**Comment by Skaruts** [comment]
In my case I suppose I need `-path`, unless I'm missing something. qbsp was complaining that it couldn't find my textures, and using `-path` fixed it. 

I'm not using wad files, just loose textures, but it seems to be working fine.

**Comment by jitspoe** [comment]
I take back what I said earlier.  I am using one command line parameter: -subdivide 0 

This isn't necessary, though.  It just reduces the amount of subdivision that happens, because we don't have the same limits that Quake does.

Also, I just updated the addon to support the BSP2 format, though you really shouldn't use that format unless needed, as it just makes larger files.  Ericw's qbsp will automatically switch to BSP2 if you go beyond the limits that can be stored in vanilla Quake BSP.

**Comment by Yagich** [comment]
> In my case I suppose I need -path, unless I'm missing something. qbsp was complaining that it couldn't find my textures, and using -path fixed it.

@Skaruts which compiler are you using? with Ericw's qbsp i don't get a `-path` option 🤔

**Comment by Skaruts** [comment]
I'm using ericw's qbsp. I got it from the description in jitspoe's video about the plugin.

https://ci.appveyor.com/project/EricWasylishen/ericw-tools/build/artifacts

**Comment by iwakura-midi** [comment]
> > In my case I suppose I need -path, unless I'm missing something. qbsp was complaining that it couldn't find my textures, and using -path fixed it.
> 
> @Skaruts which compiler are you using? with Ericw's qbsp i don't get a `-path` option 🤔

The Linux version of qbsp doesn't seem to support the path option, I use the Windows version over wine to get around it, along with a bash file to shove all the texture folders in it.
